includes/Client.hpp

```diff
@@ -18,9 +18,14 @@
 #include <sys/types.h>
 
 #include <iostream>
+#include <map>
 #include <sstream>
 #include <string>
 
+#include "../includes/Channel.hpp"
+
```

Comment:
This include is not necessary because there is Channel class declaration in this file.